src/pal/src/safecrt/strlen_s.cpp

```diff
@@ -42,7 +42,7 @@
 *
 *******************************************************************************/
 
-size_t __cdecl strnlen(const char *str, size_t maxsize)
+size_t __cdecl strnlen(const char *str, size_t maxsize) throw()
```

Comment:
I couldn't figure out why it was needing exception spec there, but that was the error.

> /home/kchoi/coreclr/src/pal/src/safecrt/strlen_s.cpp:45:16: error: 'strnlen' is missing exception specification 'throw()' [-Werror]
size_t __cdecl strnlen(const char *str, size_t maxsize)
               ^
                                                        throw()
/home/kchoi/coreclr/src/pal/inc/mbusafecrt.h:68:19: note: previous declaration is here
    extern size_t strnlen( const char* inString, size_t inMaxSize );
                  ^
1 error generated.